### PR TITLE
[BPK-3415] Calendar month heading RTL alignment

### PR DESCRIPTION
**Pull request by georgegillams**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/master/Backpack/Backpack.h)
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

File: Backpack/Calendar/Classes/BPKCalendarStickyHeader.m [hunk 1]
```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithFrame:(CGRect)frame {
     self = [super initWithFrame:frame];
 
     if (self) {
-        self.titleLabel.textAlignment = NSTextAlignmentLeft;
+        self.titleLabel.textAlignment = NSTextAlignmentNatural;
```
Comment (georgegillams) [inline Backpack/Calendar/Classes/BPKCalendarStickyHeader.m]: Before this was causing the text to appear on the left even for RTL languages
Now it appears on the left for LTR and the right for RTL

**Comment by georgegillams** [comment]
> Nice catch 🙏 `NSTextAlignmentNatural` is the default for `UILabel` and `BPKLabel` so it can just be omitted

That's what I thought, but then it goes to the center

**Comment by georgegillams** [comment]
> Nice catch 🙏 `NSTextAlignmentNatural` is the default for `UILabel` and `BPKLabel` so it can just be omitted

That's what I thought, but then it goes to the center

Simulator Screen Shot - iPhone 8 - 2019-12-16 at 10 12 09

**Comment by k0nserv** [comment]
It's inherited behaviour from the library as it turns out. Can we use the property syntax consistently namely in `configureAppearance`?

**Pull request merged**